python/cudf/cudf/core/column/string.py

```diff
@@ -689,7 +689,7 @@ def deserialize(cls, header, frames):
                 each_frame = cuda.as_cuda_array(each_frame)
             elif isinstance(each_frame, memoryview):
                 each_frame = np.asarray(each_frame)
-                each_frame = cudautils.to_device(each_frame)
+                each_frame = rmm.to_device(each_frame)
```

Comment:
Can we move this to use a `Buffer` constructor instead and then get the pointer via normal methods below?

Reply:
Yeah we could do that. Wasn't sure if this was part of the legacy string code that was getting ripped out or not.


Should add was also looking at adding a fast path in `rmm.to_device(...)` ( https://github.com/rapidsai/rmm/pull/264 ), which this would then leverage.

(Unfortunately we still need `to_device` in dask-cuda currently. Trying to figure out how to get past that.)

Reply:
The `serialize` and `deserialize` code have been updated I believe where we should use Buffers instead within them.

Reply:
Yeah it's just using `nvstrings` after this.

Reply:
Alright the latest commit should fix this. Though please let me know if not 🙂